docs/references/strategies/languages/scala/sbt.md

```diff
@@ -17,4 +19,40 @@ Directories that contain `build.sbt` files are treated as sbt projects
 ## Analysis
 
 1. Run `sbt makePom` to generate pom files
-2. Use the pom.xml maven strategy to "link together" related poms into projects, and extract a dependency graph
+
+## Strategy: `sbt dependencyTree`
+
+1. From generated pom file, identify project name via, `<name>` attribute.
+2. Perform `sbt $project/dependencyTree` and parse it's output to create dependency graph
+
+With [sbt 1.4.0 release](https://www.scala-sbt.org/1.x/docs/sbt-1.4-Release-Notes.html#sbt-dependency-graph+is+in-sourced), `dependencyTree` command is available by default. If you are using
+older sbt version, you can install following plugin: https://github.com/sbt/sbt-dependency-graph, which will also enable fossa-cli to work with `dependencyTree` task.
```

Comment:
```suggestion
1. From the generated pom file, identify the project name via the `<name>` attribute.
2. Perform `sbt $project/dependencyTree` and parse its output to create a dependency graph

With [sbt 1.4.0 release](https://www.scala-sbt.org/1.x/docs/sbt-1.4-Release-Notes.html#sbt-dependency-graph+is+in-sourced), the `dependencyTree` command is available by default. If you are using
an older sbt version, you can install the following plugin: https://github.com/sbt/sbt-dependency-graph. This will also enable fossa-cli to work with the `dependencyTree` task.
```